### PR TITLE
VS2019 build action

### PR DESCRIPTION
**Pull request by QilongTang**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Add a VS2019 build action for Dynamo which is free and maybe needed for 3rd party devs before we update our existing check (VS2017).

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

@DynamoDS/dynamo

### CODE REVIEW

File: .github/workflows/dynamoMSbuild.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - name: Build Dynamo with MSBuild
+      run: |
+        echo "***Execute ls, and you should see two folders created***"
+        ls
+        echo "***Continue with the build, Good luck developer!***"
+        cd "C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\"
+           .\MSBuild.exe $Env:GITHUB_WORKSPACE\Dynamo\src\Dynamo.All.sln
+    # look for Dynamo
+    - name: Find Dynamo
+      run: |
```
Comment (mjkkirschner) [inline .github/workflows/dynamoMSbuild.yml]: @QilongTang will this actually fail if Dynamo binaries are not created? Can we assert anything here?

Comment (QilongTang) [reply .github/workflows/dynamoMSbuild.yml]: Not free, going to use https://github.com/marketplace/actions/file-existence

Comment (mjkkirschner) [reply .github/workflows/dynamoMSbuild.yml]: @QilongTang if thats too difficult, what about just tryin to execute a DynamoCLI.exe instance?

Comment (QilongTang) [reply .github/workflows/dynamoMSbuild.yml]: Yes it will fail because bash will fail

**Comment by aparajit-pratap** [comment]
Interesting to see how these actions are written.

File: .github/workflows/dynamoMSbuild.yml (hunk 1 above)
Comment (QilongTang) [reply .github/workflows/dynamoMSbuild.yml]: @mjkkirschner To clarify, end up using bash only to reduce complexity

**Comment by QilongTang** [comment]
@mjkkirschner @aparajit-pratap I am merging because I saw some PR starting to incorporate VS2019 changes in them so I want to make sure we know the state of PRs.. Feel free to test action in the new PRs

**Pull request merged**